src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/LegacyPaymentAccountsByEmailTest.java

```diff
@@ -46,7 +46,7 @@ public void get_request_returns_correct_payment_accounts_for_user_email_address_
 
         Map<String, Object> orgResponse = professionalReferenceDataClient.findLegacyPbaAccountsByUserEmail("some@email.com");
 
-        assertThat(orgResponse.get("payment_accounts").toString().equals("PBA1234567"));
+        assertThat(orgResponse.get("paymentAccounts").toString().equals("PBA1234567"));

```

Comment:
it is good to follow Sonar comment here for not having variable name in camel context , but since FR is already consuming the response field "payment_accounts" and they will fail to get response field, so revert back and make it as existing